### PR TITLE
azure-ad-updates

### PR DESCRIPTION
**Pull request by dkim32**
# Pull Request/Issue Resolution

**Description of Change:**
New format for specific IdP documentation: 
- IdP intro 
- Requirements 
- Service Provider (SP) Initiated Login within Azure AD
- Create and Enable a Braze API Key for IdP Login (Optional)
- Note about enforcing SAML SSO 

**Reason for Change:**
There were references to contacting customer support that we had to remove. 
In addition, this aligns with the format for custom SAML SSO setup. 

Closes #**ISSUE_NUMBER_HERE**

### Is this change associated with a Braze feature/product release?
- [ ] Yes (__Feature Release Date:__)
- [ ] No

> If yes, please note the date of the feature release.

---
---

## PR Checklist
- [ ] Ensure you have completed [our CLA](https://www.braze.com/docs/cla/).
- [ ] Tag @EmilyNecciai as a reviewer when the your work is _done and ready to be reviewed for merge_.
- [ ] Consult the [Docs Text Formatting Guide](https://github.com/Appboy/success/wiki/Docs-Text-Formatting-Guide).
- [ ] Consult the [Docs Writing Style Guide & Best Practices](https://github.com/Appboy/success/wiki/Writing-Style-Guide-&-Best-Practices).
- [ ] Tag others as Reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `config/nginx.conf.erb`.

## ATTENTION: REVIEWERS OF THIS PR
- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Preview all changes in the linked Heroku environment (click `View deployment` button below, then "Docs". A `502` error just means you should refresh until you see the Docs Home page.
---
---

### CODE REVIEW

File: _docs/_user_guide/administrative/access_braze/single_sign_on/azure_ad1.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+- ![Azure_AD8]({% image_buster /assets/img/azure_ad8.png %})
+
+5. If you wish to configure the application in SP initiated mode, click `Set additional URLs` and enter a URL that combines [your Braze instance]({{ site.baseurl }}/user_guide/administrative/access_braze/braze_instances/#braze-instances) with the following pattern: `https://<SUBDOMAIN>.braze.com/sign_in`.
+- ![Azure_AD9]({% image_buster /assets/img/azure_ad9.png %})
+
+6. Braze expects the SAML assertions in a [specific format](#user-claims-configuration-format). You can manage the values of these attributes from the User Attributes section on the `Application Integration` page. On the `Set up Single Sign-On with SAML` page, click `Edit` to open the `User Attributes` dialog. Then, edit the claims [according to the proper format, shown below](#user-claims-configuration-format).
+- ![Azure_AD10]({% image_buster /assets/img/azure_ad10.png %})
```
Comment (papiya) [inline _docs/_user_guide/administrative/access_braze/single_sign_on/azure_ad1.md]: I tried to look at this file directly here - https://github.com/Appboy/braze-docs/blob/e890f5d0508d461d90f62aa5d2cff19806f1bd78/_docs/_user_guide/administrative/access_braze/single_sign_on/azure_ad1.md and I do not see any images.

File: _docs/_user_guide/administrative/access_braze/single_sign_on/azure_ad1.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+After your Account Manager has enabled SAML SSO for your account, go to `Company Settings` > `Security Settings` and toggle the SAML SSO section to `ON`.
+
+On this page, you, input:
+
+| Requirement | Details |
+|---|---|
+| `SAML Name` | This will appear as the button text on the login screen. This is typically your IdP name, like "Azure AD.” |
+| `Target URL` | This is the Login URL provided by Azure AD.|
+| `Certificate` | The `x.509` certificate is provided by your IdP. |
```
Comment (papiya) [inline _docs/_user_guide/administrative/access_braze/single_sign_on/azure_ad1.md]: Would you like to add that it is base64 encoded and in pem format. This info is available on the form as well so definitely optional.

Comment (dkim32) [reply _docs/_user_guide/administrative/access_braze/single_sign_on/azure_ad1.md]: updated to ` The `x.509` PEM encoded certificate is provided by your IdP. `

File: _docs/_user_guide/administrative/access_braze/single_sign_on/azure_ad.md [hunk 3]
```diff
@@ -9,45 +9,82 @@ page_order: 3
...
+| `Target URL` | This is the Login URL provided by Azure AD.|
+| `Certificate` | The `x.509` PEM encoded certificate is provided by your IdP. |
+
+![Enable SAML SSO]({% image_buster /assets/img/samlsso.gif %})
 
-## Step 3: Configure Braze Single Sign-On
+## Create and Enable a Braze API Key for IdP Login (Optional)
 
-Send the downloaded `Certificate (Base64)` to [Braze support]({{ site.baseurl }}/support_contact) so they can turn on your SAML SSO connection.  
+To enable IdP initiated login, you will first need to create an API Key in `Developer Settings` > `API Settings`.
```
Comment (gsong1205) [inline _docs/_user_guide/administrative/access_braze/single_sign_on/azure_ad.md]: I think it's "Developer Console"

Comment (dkim32) [reply _docs/_user_guide/administrative/access_braze/single_sign_on/azure_ad.md]: great catch. thanks!

**Pull request merged**